source/develop/release-management/features/virt/dedicated-cpus.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: Dedicated CPUs
+Category: feature
+Authors: sabusale
+---
+
+# Dedicated CPUs
+
+## Summary
+
+Once choosing dedicated CPU policy, The guest vCPUs will be strictly pinned to a set of host pCPUs (similarly to static CPU pinning). The set of pCPUs will be chosen to match the required guest CPU topology.
+
+## Owner
+
+*   Name: Saif Abu Saleh (sabusale)
+*   Email: sabusale@redhat.com
+
+
+### Tasks:
+
+
+1- Add dedicated to CPU Pinning Policy enum
+
+2- Scheduling
+
+3- Send vcpupin with cpusets and policy name in VM metadata(only when Policy is dedicated?)
+
+4- Send the policy name from vmStatic policy field
+
+5- Send the cpusets (convert from CurrentCpuPinningField form to array)
+
+6- API: Migrate
+Send cpusets additional parameter (convert from CurrentCpuPinningField form to array)
+
+7- setNumberOfCpus VDSM call
+Send cpusets additional parameter (convert from CurrentCpuPinningField form to array)
+Frontend: Add Dedicated policy dropdown to the UI
+
+
+Steps 3-7: can be done in parallel by different person
+
+
+### Scheduling:
+
+* Find host that have the VM CPU topology on the scheduling phase
+* On run phase, schedule the VM to run on that host
```

Comment:
On run phase, create the VM CPU pinning on the selected host.

Reply:
I changed the wording